src/main/java/seedu/duke/ui/TextUi.java

```diff
@@ -11,15 +11,15 @@
 
 public class TextUi {
 
-    public static final String PRINT_LISTALL_FORMAT = "|%-3s||%-15s||%-8s||%-10s||%-10s|";
-    public static final String PRINT_LIST_FORMAT =  "|%-3s||%-16s||%-12s||%-17s|";
+    public static final String PRINT_LISTALL_FORMAT = "|%-3s||%-35s||%-8s||%-10s||%-10s|";
+    public static final String PRINT_LIST_FORMAT =  "|%-3s||%-36s||%-12s||%-17s|";
 
     private final PrintStream out;
     private final Scanner in;
     private static final Logger logger = Logger.getLogger(TextUi.class.getSimpleName());
 
-    private static final String DIVIDER = "=".repeat(57);
-    private static final String SINGLE_DIVIDER = "-".repeat(57);
+    private static final String DIVIDER = "=".repeat(76);
```

Comment:
maybe a short comment to let the reader know 76 is the length of the divider or using a static final int